lib/rucio/web/rest/flaskapi/v1/common.py

```diff
@@ -103,6 +103,15 @@ def dispatch_request(self, *args, **kwargs):
                              'servers to handle. Please try again in a few minutes.'),
                     headers=headers
                 )
+            elif 'DELETED_DIDS_PK violated' in str(error):
```

Comment:
In principle this would work, but this is the wrong location to put it. The reason why the `QueuePool` error is put in the `ErrorHandlingMethodView` here is that it can happen in ANY REST call. But in this case, the `DELETED_DIDS_PK violated` only happens during registration of a did. So it should only be put into that specific endpoint. As a second point: It would be nice to put the scope/name of the DID in the error message